src/redemptions/redemption.service.ts

```diff
@@ -78,63 +87,99 @@ export class RedemptionService {
         idDetails,
       };
     }
+    const repeatedFaceWorkflowIds = this.getRepeatedFaceWorkflowIds(
+      transactionStatus.capabilities.imageChecks,
+    );
+    const multiAccountFailure = await this.multiAccountFailure(
+      userId,
+      repeatedFaceWorkflowIds,
+    );
+
     // TODO: HANDLE WARN, use decision.risk.score?
-    const failure =
-      this.livenessStatus(transactionStatus) ||
-      this.similarityStatus(transactionStatus) ||
-      this.dataChecksStatus(transactionStatus) ||
-      this.extractionStatus(transactionStatus) ||
-      this.usabilityStatus(transactionStatus);
+    const failure = multiAccountFailure || this.labelFailure(labels);
     if (failure) {
       return {
         status: KycStatus.FAILED,
         failureMessage: failure,
         idDetails,
       };
     }
```

Comment:
Even though we technically don't need `!multiAccountFailure` currently, if someone moved this code block higher up in the function, this would unexpectedly pass